File: Solve/A1/A1-018/A1-018.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d",&n);
    const char *r[9] = {"I","II","III","IV","V","VI","VII","VIII"};

    if (n < 0){
        printf("Error : Please input positive number");
    }
    else if (n < 1 || 9 < n){
        printf("Error : Out of range");
    }
    else{
        printf("%s",r[n-1]);
    }
}

File: Solve/A1/A1-018/A1-018.py
a = ["I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX"]
n = int(input())
if (1 <= n and n<=9):
    print(a[n-1])
elif (n < 0):
    print("Error : Please input positive number")
else:
    print("Error : Out of range")